docs/installation-and-operation/remote-sync.md

```diff
@@ -20,91 +30,50 @@ You work with Remote Sync through special collections called "synced collections
+**Branch-based workflow**: In Development mode, you can create branches, switch between them, and commit changes — all from the Metabase UI. This enables feature-branch workflows where you develop on a branch, open a pull request for review, and merge to production.
 
-### Remote Sync vs. Serialization
+### Remote Sync excludes table metadata
 
-Remote Sync uses the same underlying serialization format as the [Metabase CLI serialization feature](./serialization.md), but serves a different purpose:
+Remote Sync only syncs the collection (and its subcollections). This means any customizations you make to [table metadata](../data-modeling/metadata-editing.md) in your development instance—such as changing a column's display type from "Unix timestamp" to "Date", adding column descriptions, or adjusting visibility—will not automatically sync to production.
 
-- **Remote Sync**: Designed for adding new content to a read-only production environment through a Git-based workflow. Best when you want a centralized team creating content in development and deploying it to production automatically.
-- **Serialization**: Command-line tool for exporting and importing Metabase content. Best for migrating existing content, complex multi-environment setups, and scenarios requiring full control over what gets exported and imported.
+If you need to import and export table metadata as well, check out [serialization](./serialization.md).
 
 ## Setting up Remote Sync
 
 You'll need to be an admin to set up Remote Sync.
 
 1. [Set up a repository to store your content](#1-set-up-a-repository-to-store-your-content)
 2. [Create a personal access token for development](#2-create-a-personal-access-token-for-development)
-3. [Understand Development and Production modes](#3-understand-development-and-production-modes)
-4. [Connect your development Metabase to your repository](#4-connect-your-development-metabase-to-your-repository)
-5. [Add an item to your synced collection](#5-add-an-item-to-your-synced-collection)
-6. [Push your changes to your repository](#6-push-your-changes-to-your-repository)
-7. [Create a personal access token for production](#7-create-a-personal-access-token-for-production)
-8. [Connect your production Metabase to your repository](#8-connect-your-production-metabase-to-your-repository)
+3. [Connect your development Metabase to your repository](#3-connect-your-development-metabase-to-your-repository)
+4. [Add an item to your synced collection](#4-add-an-item-to-your-synced-collection)
+5. [Push your changes to your repository](#5-push-your-changes-to-your-repository)
+6. [Create a personal access token for production](#6-create-a-personal-access-token-for-production)
+7. [Connect your production Metabase to your repository](#7-connect-your-production-metabase-to-your-repository)
 
-## 1. Set up a repository to store your content
+### 1. Set up a repository to store your content
```

Comment:
I kind of liked it as H2 so i could see the steps immediately when opening the article. but I guess it also makes ToC seem very long so i'm ambivalent